site/content/docs/4.3/getting-started/browsers-devices.md

```diff
@@ -8,7 +8,7 @@ toc: true
 
 ## Supported browsers
 
-Bootstrap supports the **latest, stable releases** of all major browsers and platforms. On Windows, **we support Internet Explorer 11 / Microsoft Edge**.
+Bootstrap supports the **latest, stable releases** of all major browsers and platforms. On Windows, **we support Microsoft Edge**.
```

Comment:
Maybe we should remove the Windows-specific sentence since it's covered by the previous one?

Reply:
Updated this: https://github.com/twbs/bootstrap/pull/30377/commits/642ff6b15b3da1b681ad5ce5713e1e7ab4088894